MOLFCMClient/MOLFCMClient.m

```diff
@@ -82,19 +91,23 @@ @implementation MOLFCMClient
 #pragma mark init/dealloc methods
 
 - (instancetype)initWithFCMToken:(NSString *)FCMToken
+                            host:(NSString *)host
+                 connectDelayMax:(uint32_t)connectDelayMax
+                      backoffMax:(uint32_t)backoffMax
+                      fatalCodes:(NSArray<NSNumber *> *)fatalCodes
```

Comment:
qq: why is this overrideable? presumably as a client of FCM this class should know best about status codes.

Reply:
You would think. This is mainly so we can change them without having to do a release here.